packages/corvid-cli/src/utils/electron.js

```diff
@@ -29,68 +31,62 @@ const beforeCloseDialogParams = {
 const runningProcesses = [];
 
 function launch(file, options = {}, callbacks = {}, args = []) {
-  options.env = {
-    ...process.env,
-    ...options.env,
-    FORCE_COLOR: "yes"
-  };
-
-  const cp = childProcess.spawn(
+  const childElectronProcess = childProcess.spawn(
     electron,
     [path.resolve(path.join(file)), ...args],
     {
-      ...options
+      ...options,
+      env: {
+        ...process.env,
+        ...options.env,
+        FORCE_COLOR: "yes"
+      },
+      stdio: ["ignore", "inherit", "pipe", "ipc"]
     }
   );
-  runningProcesses.push(cp);
+  runningProcesses.push(childElectronProcess);
 
   function isJunk(data) {
     const text = data.toString();
     return text.includes("Could not instantiate: ProductRegistryImpl.Registry");
   }
 
-  return new Promise((resolve, reject) => {
-    const messages = [];
+  childElectronProcess.stderr.on("data", function(data) {
```

Comment:
@tomenden i changed stderr to be a pipe like before, so that we can filter electron junk errors from the user's screen. we can still debug those errors via the logger when running in development mode.
wdyt?